tests/unit/test_call_endpoint.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2019 Sophos Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.
+# You may obtain a copy of the License at:  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and limitations under the
+# License.
+#
+
+"""
+ Unit tests for Sophos SIEM.
+ Requirements
+  - Python 2.7 (ActivePython recommended on Windows)
+  - Python 3.5 (ActivePython recommended on Windows)
+
+"""
+
+import os
+import unittest
+import siem
+
+import mock
+import json
+
+
+class CallEndPointTest(unittest.TestCase):
+
+    @mock.patch('siem.config.Token')
+    @mock.patch('siem.store_state')
+    def test_data_stream_in_event(self,
+                                  mock_cf_token,
+                                  mock_store_state
+                                  ):
```

Comment:
[ISSUE] Patch annotation is read bottom-up and provided as arguments left to right. Either change the order of the annotations or change the order of the arguments.

Reply:
Behavior documented [here](https://docs.python.org/3/library/unittest.mock.html#nesting-patch-decorators).

Reply:
Done